dev/firely/docker-compose.mssqlserver.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+
+  firely:
+    depends_on:
+      - db
+    environment:
+      - VONK_Repository=SQL
+      - VONK_SqlDbOptions:ConnectionString=Initial Catalog=VonkData;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+      - VONK_SqlDbOptions:SchemaName=vonk
+      - VONK_SqlDbOptions:AutoUpdateDatabase=true
+      - VONK_SqlDbOptions:AutoUpdateConnectionString=Initial Catalog=VonkData;Data Source=db,1433;User ID=sa;Password=SQLServerStrong(!)Password*
+
+  db:
+    image: mcr.microsoft.com/mssql/server
```

Comment:
Interesting - do we need a license for this @daniellrgn ?

Reply:
Yes, though I've been using evaluation licenses and updating them each week, I haven't run into a limit on them yet.

Reply:
Misunderstood in my previous reply. I haven't actually used this mssql configuration yet, just sqlite and mongodb so far.

Reply:
By default, the MSSQL image runs with the `Developer` license, which doesn't require any registration/activation etc

>MSSQL_PID is the Product ID (PID) or Edition that the container will run with. Acceptable values:
>Developer : This will run the container using the Developer Edition (this is the default if no MSSQL_PID environment variable is supplied)

See the [License](https://hub.docker.com/_/microsoft-mssql-server) section for more info